stytch/src/main/kotlin/com/stytch/java/consumer/models/sessions/Sessions.kt

```diff
@@ -130,6 +130,9 @@ public enum class AuthenticationFactorDeliveryMethod {
 
     @Json(name = "oauth_exchange_hubspot")
     OAUTH_EXCHANGE_HUBSPOT,
+
+    @Json(name = "oauth_exchange_github")
+    OAUTH_EXCHANGE_GITHUB,
```

Comment:
Is it an issue to have some of the GitHub type declarations in here even though it's not live yet?